hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropBase.java

```diff
@@ -64,11 +65,12 @@ public class TokenAirdropBase {
     protected static final String FT_WITH_HBAR_FIXED_FEE = "fungibleTokenWithHbarCustomFee";
     protected static final String FT_WITH_HTS_FIXED_FEE = "fungibleTokenWithHtsCustomFee";
     protected static final String FT_WITH_FRACTIONAL_FEE = "fungibleTokenWithFractionalFee";
+    protected static final String FT_WITH_FRACTIONAL_FEE_NET_OF_TRANSFERS = "ftWithFractionalFeeNetOfTransfers";
     protected static final String NFT_WITH_HTS_FIXED_FEE = "NftWithHtsFixedFee";
     protected static final String NFT_WITH_ROYALTY_FEE = "NftWithRoyaltyFee";
     protected static final String DENOM_TOKEN = "denomToken";
     protected static final String HTS_COLLECTOR = "htsCollector";
-    protected static final String HTS_COLLECTOR2 = "htsCollector2";
+    // protected static final String HTS_COLLECTOR2 = "htsCollector2";
```

Comment:
We should delete this if we are not using it.

Reply:
Done